src/clr-angular/utils/animations/expandable/expandable.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Component, ElementRef, HostBinding, HostListener, Input } from '@angular/core';
+
+import { animate, style, transition, trigger } from '@angular/animations';
+import { DomAdapter } from '../../dom-adapter/dom-adapter';
+
+@Component({
+  selector: 'clr-expandable',
+  template: `
+    <ng-content></ng-content>
+  `,
+  styles: [
+    `
+    :host {
+      display: block;
+      overflow: hidden;
+    }
+  `,
+  ],
+  host: {
+    class: 'expandable',
```

Comment:
The same reason as above would apply here too. So I am kinda questioning the existence of this class.

Reply:
Removed.